docs/source/trading-calendars.rst

```diff
@@ -0,0 +1,172 @@
+
+Now we'll take a look at the London Stock Exchange Calendar :class:`~zipline.utils.calendars.exchange_calendar_lse.LSEExchangeCalendar` as an example below:
+
+.. code-block:: python
+
+  class LSEExchangeCalendar(TradingCalendar):
+    """
+    Exchange calendar for the London Stock Exchange
+
+    Open Time: 8:00 AM, GMT
+    Close Time: 4:30 PM, GMT
+
+    Regularly-Observed Holidays:
+      - New Years Day (observed on first business day on/after)
+      - Good Friday
+      - Easter Monday
+      - Early May Bank Holiday (first Monday in May)
+      - Spring Bank Holiday (last Monday in May)
+      - Summer Bank Holiday (last Monday in May)
+      - Christmas Day
+      - Dec. 27th (if Christmas is on a weekend)
+      - Boxing Day
+      - Dec. 28th (if Boxing Day is on a weekend)
+    """
+
+    @property
+    def name(self):
+      return "LSE"
+
+    @property
+    def tz(self):
+      return timezone('Europe/London')
+
+    @property
+    def open_time(self):
+      return time(8, 1)
+
+    @property
+    def close_time(self):
+      return time(16, 30)
+
+    @property
+    def regular_holidays(self):
+      return HolidayCalendar([
+        LSENewYearsDay,
+        GoodFriday,
+        EasterMonday,
+        MayBank,
+        SpringBank,
+        SummerBank,
+        Christmas,
+        WeekendChristmas,
+        BoxingDay,
+        WeekendBoxingDay
+      ])
+
+You can create the ``Holiday`` objects mentioned in ``def regular_holidays(self)` through the `pandas <http://pandas.pydata.org/pandas-docs/stable/>`__ module, ``pandas.tseries.holiday.Holiday``, and also take a look at the `LSEExchangeCalendar <https://github.com/quantopian/zipline/blob/master/zipline/utils/calendars/exchange_calendar_lse.py>`__ code as an example.
+
+
+Building a Custom Trading Calendar
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we'll build our own custom trading calendar. This calendar will be used for trading assets that can be traded on a 24/7 exchange calendar. This means that it will be open on Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, and Sunday, and the exchange will open at 12AM and close at 11:59PM. The timezone which we'll use is UTC.
+
+First we'll start off by importing some modules that will be useful to us.
+
+.. code-block:: python
+
+  # for setting our open and close times
+  from datetime import time
+  # for setting our start and end sessions
+  import pandas as pd
+  # for setting which days of the week we trade on
+  from pandas.tseries.offsets import CustomBusinessDay
+  # for setting our timezone
+  from pytz import timezone
+
+  # for creating and registering our calendar
+  from zipline.utils.calendars import (
+    register_calendar,
+    TradingCalendar
+  )
+  from zipline.utils.memoize import lazyval
+
+
+And now we'll actually build this calendar, which we'll call ``TFSExchangeCalendar``:
+
+.. code-block:: python
+
+  class TFSExchangeCalendar(TradingCalendar):
+    """
```

Comment:
This is super minor, but should we add periods to the end of each of these docstrings?